packages/mirage-solo5/mirage-solo5.0.5.0+dune/opam

```diff
@@ -23,6 +23,7 @@ depends: [
   "lwt" {>= "2.4.3"}
   "ocaml-freestanding" {>= "0.4.4"}
   "mirage-os-shim"
+  "metrics"
```

Comment:
This is unrelated. Can you submit 2 PRs?

Reply:
Tool late